rfcs/text/0000-alternative-planning.md

```diff
@@ -0,0 +1,308 @@
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Roadmaps and Planning - an alternative perspective
+
+## Summary
+
+This informational RFC describes problems with the roadmaps and planning RFC where it concerns software
+development. It then proposes alternative approaches. This is the author's own point of view.
+
+Please note: the list of reviewers chosen is somewhat arbitrary based on my memory of who concerns themselves with this
+stuff. And Github limits the number to 15. Apologies if I left you out - guaranteed I'd like you to read this even/esepcially 
+if you strongly disagree.
+
+## Author(s)
+
+[Justin Clark-Casey](mailto:justincc@ebi.ac.uk)
+
+## Shepherd
+***Leave this blank.*** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Roadmaps and planning are the crankshaft of the DCP. If we don't create a crankshaft suitable for the engine of development 
+then the DCP will run inefficiently or not at all. I'm concerned that the proposed process for creating these roadmaps and plans, as 
+described in the Roadmaps and Planning RFC and as an evolution of the current process, does not fit DCP organizational and development realities.
+
+This RFC describes the problems and promotes alternative solutions, as a response to being asked to write these up. It's 
+nowhere near as comprehensive, well-written or referenced as I would like - unfortunately I've had many other priorities
+to attend to. Nonetheless, I hope it can help spur constructive discussion on these topics. It is the author's personal
+point of view as a technical lead and someone with long experience in highly distributed open-source projects that require co-operation
+between peer organizations and individuals.
+
+## Detailed Design
+
+From my understanding, the roadmaps and planning RFC details a 70 day process where scientists, project leads, product owners, user experience
+experts and technical leads create a roadmap consisting of themes, each theme having one of more objectives. These 
+objectives are further described by user stories. Each objective is represented in the DCP Zenhub board as an epic with
+user stories represented by an item attached to the epic. Further items can be created as spikes to spend time clarifying
+the user stories and/or performing design work prior to implementation. Each item is assigned a milestone where there are
+3 milestones of equal length in a quarter.
+
+In my experience of the existing planning process almost all spikes are cross-component issues. DCP culture practically
+ mandates that these require an RFC. An RFC
+takes a minimum of 3 weeks from inception to approval, though realistically RFCs of any complexity require considerably
+more time to understand the problem and align with other planned and existing technical work.
+
```

Comment:
This fits with my observations.   The RFC process is very slow-moving, however, there does need to be some why to formally communicate, as one component can't change things out from another.  I don't really have any good ideas on how to improve it.

Reply:
@maniarathi and @NoopDog are currently working on an update to the RFC process, but it's not ready for community review. Part of the intention is to streamline the review process.